qiskit/schemas/test_schemas.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you move this file to `tools/`, adjusting the path to the schemas used inside it accordingly?

Reply:
Does it make more sense to just make this part of the test scripts?

Reply:
Sure, `test/schemas` (or similar) is also an option - as long as it is not part of `qiskit/*` (since that folder is meant to be the folder where the source code for the SDK _packages_ lives, and having `.py` files that are not part of it might cause issues with the tooling), and also not part of `test/python` (since currently the file does not follow `unittest` conventions), sounds fine!